RELEASE_NOTES.md

```diff
@@ -299,6 +299,39 @@ Our underlying SSH implementation has been updated to support the new ed25519 SS
 
 Chef Solo's `--delete-entire-chef-repo` option has been extended to work in Local Mode as well. Be warned that this flag does exactly what it states, and when used incorrectly, can result in loss of work.
 
+### sysctl now accepts a comments parameter
+
+The sysctl resource has been updated to allow additional (optional) comments. Comments can be passed as an array or a string. Any comments provided are prefixed with '#' signs and precede the sysctl setting in generated files.
```

Comment:
I think the first sentence would benefit from removing the parenthesis, so that:
> The sysctl resource has been updated to allow additional**,** optional comments.
You should use a comma as these adjectives are coordinate adjectives.

Reply:
Thought I'd done these. Apologies for the delay.
I've actually reworded it to remove that part.